multinode-demo/leader.sh

```diff
@@ -6,5 +6,13 @@ export RUST_LOG=${RUST_LOG:-solana=info}
 set -x
 [[ $(uname) = Linux ]] && sudo sysctl -w net.core.rmem_max=26214400
 
+IPADDR="$(ifconfig  | grep 'inet addr:'| grep -v '127.0.0.1' | cut -d: -f2 | awk '{ print $1}')"
+
+if [ -z "$IPADDR" ]; then
```

Comment:
use bash's builtin [[ instead of [ (which is a program)
doing so saves you having to quote IPADDR

Reply:
Good point. will do